docs/StackNavigation-Setup.md

```diff
@@ -34,7 +34,7 @@
      { typeof(MainPageViewModel), typeof(MainPage) }
    };
    ```
-1. From a blank UWP application, create a `FrameStackNavigator` using the predefined `rootFrame` in the `OnLaunched` method of `App.xaml.cs`.
+1. From a blank UWP (Legacy) application, create a `FrameStackNavigator` using the predefined `rootFrame` in the `OnLaunched` method of `App.xaml.cs`.
```

Comment:
Couldn't we just mention WinUI instead of "UWP (Legacy)"?
Is this setup really specific to UWP?